melissa/profile_populator.py

```diff
@@ -94,7 +94,7 @@ def empty(variable):
  or <ENTER> for a search based on the name\
  of the city you live in: ')
     if empty(city_code):
-        city_list = pywapi.get_loc_id_from_weather_com(city_name)
+        city_list = pywapi.get_loc_id_from_weather_com(unicode(city_name))
```

Comment:
i think the readme can be left out in other pull request and keep the this warning fix alone. i don't know about other platform/python shell user, but this make at least ipython user not see this warning anymore. but someone have to keep this in mind, if this project have to run with python 3 (later) #122.

i just merge it only that single line of code, without readme changes.